data/virt_autotest/guest_params_xml_files/slem_5_5_64_kvm_hvm_x86_64.xml

```diff
@@ -84,7 +84,7 @@
     <guest_autoconsole>text</guest_autoconsole>
     <guest_noreboot></guest_noreboot>
     <guest_default_target>graphical</guest_default_target>
-    <guest_do_registration>true</guest_do_registration>
+    <guest_do_registration>false</guest_do_registration>
```

Comment:
Why no registration? I remember scc problem is fixed.

Reply:
due to bsc#1213947?

Reply:
Can not register with scc.suse.com at the moment.

Reply:
Aha, right. Would you please add a comment with the bug ?

Reply:
> due to bsc#1213947?

Yes. The same error with https://bugzilla.suse.com/show_bug.cgi?id=1213947

Reply:
> Aha, right. Would you please add a comment with the bug ?

Where to add ? I do not think it is very necessary. Registration with scc.suse.com always bit later. I will switch to do registration later.

Reply:
In this file, with format " <!-- comment -->".